src/Services/SwaggerService.php

```diff
@@ -562,11 +562,13 @@ public function saveConsume()
 
     public function saveTags()
     {
-        $tagIndex = 1;
+        $globalPrefix = config('auto-doc.global_prefix');
 
         $explodedUri = explode('/', $this->uri);
 
-        $tag = Arr::get($explodedUri, $tagIndex);
+        $tag = (!empty($globalPrefix) && "/{$explodedUri[1]}" === $globalPrefix)
+            ? Arr::get($explodedUri, 2)
+            : Arr::get($explodedUri, 1);
```

Comment:
please to implement something like this

```suggestion
        $tag = array_shift($explodedUri);

        if ($globalPrefix === $tag)) {
            $tag = array_shift($explodedUri);
        }
```